### ISSUE TITLE
Samples and Flash

### ISSUE THREAD

**Issue by RAWJUNGLE** [democloid/picoTracker#21]
@maks @democloid  

Pico-tracker, works correctly only with samples, mono 16 bit 22050Hz. all 8 tracks work well together with long and short samples at the same time. 

16 stereo 44100Hz - do not work correctly, sometimes they do not turn on.

Samples - https://discord.com/channels/1166483348634480650/1167241783026139186/1176193011521163344

I checked, for some reason you can't work with samples in more than 3 tracks. They just don't work. 

Track 1 - vocals 
Track 2 - drums 
Track 3 - pad

and then the samples don't work.

I tried to make them different lengths, sometimes 2 tracks work, sometimes three. 

Now I tried to upload many parts 

All samples 8 secs long

4 bass files  
4 pad files 
1 drum file 
4 vocal files 

bass/pad/vocals/drums - each file has a size of 353kb

Short samples work fine. in 8 tracks / 16 instruments
But if you add samples of 8 seconds or more, problems occur. 
There is enough memory though, the samples come out to 5 megabytes. 

Project - https://discord.com/channels/1166483348634480650/1167241783026139186/1176193494587547648

My Pico 16mb Black. All samples mono 16 bit 22050Hz.

**Comment by democloid** [comment]
Did some testing on what works and what fails. I confirmed that the 27s mono 16bit 22050Hz file seems to work fine (although with some distortion?) but the stereo 16bit 44100Hz doesn't, never finishes copying to flash.

I used the failing file to create shorter samples, of length: 15s, 10s, 5s and 2s (file sizes: 2.52MiB, 1.68MiB, 861KiB and 345KiB)
15s and 10s also fail to load, 5s and 2s load fine and work as expected, so it's not a problem with stereo or 44100Hz files.

Will need to investigate further.
